extensions/azurehybridtoolkit/yarn.lock

```diff
@@ -0,0 +1,2392 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+ajv@^6.12.3:
```

Comment:
Is this accurate? There seems to be an awful lot of packages here for something that has basically no dependencies.